Sources/Concrete/ViewModel/HitsViewModel.swift

```diff
@@ -12,31 +12,39 @@ import InstantSearchCore
 /// ViewModel - View: HitsViewModelDelegate.
 ///
 /// ViewModel - Searcher: SearchableViewModel, ResultingDelegate, ResettableDelegate.
-internal class HitsViewModel: HitsViewModelDelegate, SearchableViewModel {
+public class HitsViewModel: HitsViewModelDelegate, SearchableIndexViewModel {
     
     // MARK: - Properties
     
-    var hitsPerPage: UInt {
+    public var indexId: String {
```

Comment:
Same question, why separate the logic of id and name instead of having a struct with both ?

Reply:
Indeed this is something I should fix. I had to separate them in views so that they can be shown as IBInspectable. But the VMs should merge both in the SearcherId class. I'll fix.